databricks_cli/clusters/cli.py

```diff
@@ -50,6 +52,29 @@ def create_cli(api_client, json_file, json):
     """
     json_cli_base(json_file, json, lambda json: ClusterApi(api_client).create_cluster(json))
 
+@click.command(context_settings=CONTEXT_SETTINGS)
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON request to POST to /api/2.0/clusters/edit.')
+@click.option('--json', default=None, type=JsonClickType(),
+              help=JsonClickType.help('/api/2.0/clusters/edit'))
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def edit_cli(api_client, json_file, json):
+    """
+    Edits a Databricks cluster.
+
+    The specification for the request json can be found at
+    https://docs.databricks.com/api/latest/clusters.html#edit
+    """
+    if not bool(json_file) ^ bool(json):
```

Comment:
does the `json_cli_base` helper wwork here?

Reply:
I took that from jobs edit call https://github.com/databricks/databricks-cli/blob/master/databricks_cli/jobs/cli.py#L80, since cluster edit doesn't have a response using `json_cli_base` seemed odd since it echoed `{}` to the console